plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -171,7 +171,10 @@ func (s *SelectorSpread) CalculateSpreadPriority(pod *v1.Pod, nodeNameToInfo map
 		if haveZones {
 			zoneId := utilnode.GetZoneKey(node)
 			if zoneId != "" {
-				zoneScore := float64(schedulerapi.MaxPriority) * ((maxCountByZone - countsByZone[zoneId]) / maxCountByZone)
```

Comment:
I guess an alternative approach would have been to initialize `maxCountByZone := float64(1)` and leave the this line as it was before, but this is fine too.

Reply:
Nice approach! But I prefer the current one that I feel is more readable. Thanks for review.